src/Okta.Sdk/Generated/IApplicationsClient.Generated.cs

```diff
@@ -96,56 +96,6 @@ public partial interface IApplicationsClient
         /// <returns>The <see cref="ICsr"/> response.</returns>
         Task<ICsr> GetCsrForApplicationAsync(string appId, string csrId, CancellationToken cancellationToken = default(CancellationToken));
 
-        /// <summary>
-        /// 
-        /// </summary>
```

Comment:
Question: Where's the code that skips these methods from generation? I think the errata file is missing in this PR.

Reply:
I changed the errata file in the previous set of changes for content-type. https://github.com/okta/okta-sdk-dotnet/pull/392/files#diff-f6381e7203bc042b528cd2c0cf01e7b5